eslint/eslintrc.yml

```diff
@@ -23,15 +23,21 @@ globals:
 
 extends:
     # Extend the base Body Labs style, which is pulled in via peerDependencies.
-    "../../bodylabs-javascript-style/eslint/eslintrc-shared.yml"
+    - '../../bodylabs-javascript-style/eslint/eslintrc-shared.yml'
 
-rules:
-    # 2 means error, 1 means warning, 0 means ignore.
+    # Pull in https://github.com/yannickcr/eslint-plugin-react
+    - 'plugin:react/recommended'
 
+rules:
     # JSX.
-    # eslint-plugin-react is required
-    # https://github.com/yannickcr/eslint-plugin-react
-    jsx-quotes: 2
+    jsx-quotes: 'error'
+
+    # More JSX, from https://github.com/yannickcr/eslint-plugin-react
+    react/jsx-curly-spacing: ['error', 'always']
+    react/jsx-equals-spacing: ['error', 'never']
+    react/jsx-indent: 'error'
```

Comment:
Can we add [`react/jsx-indent-props: 'error'`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-indent-props.md) as well? Everyone always has different styles for organizing their props once they go past one line and enforcing consistency there would be really nice.